src/v2/Apps/Consign/ConsignApp.tsx

```diff
@@ -27,11 +27,12 @@ const ConsignApp = props => {
       <>
         <ConsignMeta />
 
-        <AppContainer>
+        <AppContainer maxWidth="100%">
+          <Header />
+          <GetPriceEstimate />
```

Comment:
Stacking completed items here, and either way I think that this will be a full-bleed page